nest-forms-backend/test/fixtures/auth/user-fixture-factory.ts

```diff
@@ -0,0 +1,321 @@
+import { UnauthorizedException } from '@nestjs/common'
+import { TestingModuleBuilder } from '@nestjs/testing'
+import {
+  randCompanyName,
+  randEmail,
+  randFirstName,
+  randLastName,
+} from '@ngneat/falso'
+import jwt from 'jsonwebtoken'
+import {
+  UserOfficialCorrespondenceChannelEnum,
+  UserVerifyStateCognitoTierEnum,
+  UserVerifyStateTypeEnum,
+} from 'openapi-clients/city-account'
+import { v4 as uuidv4 } from 'uuid'
+
+import { CityAccountUserService } from '../../../src/auth-v2/services/city-account-user.service'
+import { CognitoGuestIdentityService } from '../../../src/auth-v2/services/cognito-guest-identity.service'
+import { CognitoJwtVerifyService } from '../../../src/auth-v2/services/cognito-jwt-verify.service'
+import { CognitoUserService } from '../../../src/auth-v2/services/cognito-user.service'
+import { AuthUser, GuestUser, UserType } from '../../../src/auth-v2/types/user'
+
+export type GuestFixtureUser = {
+  identityId: string
+  headers: {
+    'X-Cognito-Guest-Identity-Id': string
+  }
+  user: GuestUser
+}
+export type AuthFixtureUser = {
+  sub: string
+  headers: {
+    Authorization: string
+  }
+  user: AuthUser
+}
+
+export type FixtureUser = GuestFixtureUser | AuthFixtureUser
+
+const getRandomGuestIdentityId = () => `eu-central-1:${uuidv4()}`
```

Comment:
I know it is a test, but we already have an environmental variable for `eu-central-1`, shouldn't it be used? Or is it important this is different regardless of the content of the variable?